templates/tableau-cluster-linux.template

```diff
@@ -406,8 +406,8 @@
         },
         "DefaultConfiguration": {
             "InstallationConfig": {
-                "TableauServerInstallerOnCentos": "https://s3-us-west-2.amazonaws.com/tableau-quickstart/tableau-server.x86_64.rpm",
-                "TableauServerInstallerOnUbuntu": "https://s3-us-west-2.amazonaws.com/tableau-quickstart/tableau-server_amd64.deb",
+                "TableauServerInstallerOnCentos": "https://s3-us-west-2.amazonaws.com/tableau-quickstart/tableau-server-2018-1-1.x86_64.rpm",
```

Comment:
We should not be changing the installer name; otherwise we will need to update the templates every quarter.  I usually just drop the new installers and rename them to what the templates expect.